src/pydata_sphinx_theme/assets/styles/content/_toctree.scss

```diff
@@ -0,0 +1,33 @@
+/**
+ * Style the toctree component in pages (avoid modifying the navbars)
+ */
+.toctree-wrapper {
+  p.caption {
+    font-size: 1.5em;
+    margin-bottom: 0em;
+  }
+
+  & > ul {
+    padding-left: 0;
+  }
+
+  li[class^="toctree-l"] {
+    list-style: none;
+    margin-bottom: 0.5em;
```

Comment:
```suggestion
    margin-bottom: 0.2em;
```

Reply:
For some reason having at .5 was causing the `l1` toctree list items to be slightly differently-spaced, which was a bit unsettling...I think this would fix it